addon/components/base/bs-form/element.js

```diff
@@ -139,6 +139,11 @@ const nonDefaultLayouts = A([
  {{/bs-form}}
  ```
 
+ If yu are using the custom control quite often, you should consider writing an integration plugin like
```

Comment:
I hate to be pedantic but second word has a typo :)
*you

Reply:
Thanks for noticing. It's fixed.